### ISSUE TITLE
[Discussion] Topic discussion for team project

### ISSUE THREAD

**Issue by leemgs** [skkuse-adv/2019Fall_team2#3]
According to TA, we have to choose on among the below four items for our team project.

## Project Scope 

- [ ] 1. 라이브 커머스 Live Commerce: CJmall 쇼크 Live, Grip
- [ ] 2. 중고 거래 Used deal: 당근 마켓
- [x] 3. 개인화 Personalization: 핏츠 Fitts, 카카오스타일 Kakao Style, 롯데 홈쇼핑 Lotte Home shopping
- [ ] 4. 상점 랭킹 Store Ranking: Taobao, Wish 

## Reference
* i-campus Notice:  
  * No. 10, Title: Team project concept
  * webpage: http://icampus.ac.kr/front/study/DataAction.do?method=list&bdotDiv=01#
  * File: Software Architectural Design_Team project concept.pdf
    - [Software Architectural Design_Team project concept.pdf](https://github.com/skkuse-adv/2019Fall_team2/files/3696269/Software.Architectural.Design_Team.project.concept.pdf)
    - [Software Architectural Design_Team_Project_Instructions.pdf](https://github.com/skkuse-adv/2019Fall_team2/files/3696270/Software.Architectural.Design_Team_Project_Instructions.pdf)

**Comment by leemgs** [comment]
@beandkay  @JangHanbin @ajid951125 PTAL. I want you to post your idea freely.

**Comment by beandkay** [comment]
In my opinion, I would like to go with the **Kakao Style** idea or **Fitts**. We can build some app that interesting. It is just from myself because I don't like other product intuitions.

**Comment by leemgs** [comment]
> In my opinion, I would like to go with the Kakao Style idea or Fitts.

If we do not need to implement the proposal, I am okay if we decide anyone among the four topics. 😄 

* Kakao Style - https://style.kakao.com/main/hot
* Fitts - https://fitts.co.kr/

**Comment by leemgs** [comment]
To All, 

* http://icampus.ac.kr/front/study/DataAction.do?method=view&lmsBdotSeq=2988635&lmsBlbdId=4
The professor announced the entire schedule for the team project as following: 

```bash

The below is the activities and work products of team projects you should do this semester and the related schedule.

 

(1) Presentation on target system analysis: 10/17

- Choose a target system among some candidates

(ex. Target system: Live commerce platform based on streaming service, such as TAOBAO, Aliexpress LIVE, and so on)


- See the file provided earlier

- Presentation content (name, system configuration, main functions, reason for selection, related existing system​s)

 

(2) Generation of requirement specification through the reverse engineering: 10/31 

(3) Generation of the architecture of the target system: 11/14

(4) Creating the architecture model with the requirement specification​ based on ADD(Attribute Driven Design): 12/12

In summary, this team project is relatively simple. 
You just analyze an existing system and find out an architectural model 
through reverse engineering, and then try to create the architectural model 
of the target system through ADD technique again. The two results might be similar, 
but may be different.  
```

**Comment by leemgs** [comment]
> (1) Presentation on target system analysis: 10/17

* Team collaboration for writing our proposal together.
   * Online meeting: 2019-Oct-13 14:00 ~ 16:00
   * Participants: All (4 members: @beandkay @JangHanbin @ajid951125)
   * How to : 
      * GitHub (2019Fall_team2)
      * KakaoTalk (using the "group call" function)
      * MS Office  PPT document (MS Office 365)
   * File: [Presentation on target system analysis (pptx)](https://o365skku-my.sharepoint.com/:p:/g/personal/binhnk_o365_skku_edu/EQ4JOf1J5YdIvvYGXFxy1cEBTLtlNE7qqxOwTwPY_rCLYQ?rtime=nmccuThO10g)
